lib/src/model/robotoff_nutrient_extraction.dart

```diff
@@ -0,0 +1,88 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:openfoodfacts/src/model/per_size.dart';
+import 'package:openfoodfacts/src/model/robotoff_nutrient_extraction_annotation.dart';
+import '../interface/json_object.dart';
+import 'nutrient.dart';
+import 'robotoff_nutrient_extraction_insight.dart';
+
+part 'robotoff_nutrient_extraction.g.dart';
+
+@JsonSerializable()
+class RobotoffNutrientExtractionResult extends JsonObject {
+  final String? status;
+  final int? count;
+  final List<RobotoffNutrientExtractionInsight>? insights;
+
+  const RobotoffNutrientExtractionResult({
+    this.status,
+    this.count,
+    this.insights,
+  });
+
+  RobotoffNutrientExtractionInsight? get getLatestInsights {
```

Comment:
I don't find it very elegant to compute this again and again.
I'd prefer if you could compute that only once.
Perhaps with a bool private flag? That would prevent you from using a `const` class, which isn't a big loss, IMHO.
Please ignore this comment if you cannot find quickly a clean solution.